lib/generators/declare_schema/support/model.rb

```diff
@@ -4,6 +4,39 @@
 
 module DeclareSchema
   module Support
+    class IndentedBuffer
```

Comment:
I got tired of fighting ERB failures on travis (mystery errors inside the ruby runtime) that didn't reproduce locally. So I tossed that ERB and wrote this tiny class to handle the generated formatting. The big plus here is that it automatically produces matching indentation.